### PR TITLE
Instruct mergify to recreate dependabot PRs if they have merge conflicts

### PR DESCRIPTION
**Pull request by thomaseizinger**

### CODE REVIEW

File: .mergify.yml [hunk 1]
```diff
@@ -17,9 +17,15 @@ pull_request_rules:
       - label!=blocked
       - label!=no-mergify
       - head~=^(?!release.*).*$
+  - name: recreate dependabot PRs with merge conflicts
+    conditions:
+      - "status-success=ci/circleci: debug-build-test"
+      - "status-success=license/cla"
```
Comment (D4nte) [inline .mergify.yml]: Maybe remove this one as we don't care if licence service is down, we should still recreate.

Comment (thomaseizinger) [reply .mergify.yml]: Fixed!

**Comment by thomaseizinger** [comment]
bors retry

**Pull request merged**